examples/FailingTests.elm

```diff
@@ -86,4 +87,31 @@ testView =
                         |> Query.find [ tag "ul" ]
                         |> Query.findAll [ tag "li" ]
                         |> Query.each (Query.has [ classes [ "list-item", "themed" ] ])
+            , test "expect first a to send SomeMsg onClick" <|
+                \() ->
+                    output
+                        |> Query.findAll [ tag "a" ]
+                        |> Query.first
+                        |> Events.msgFor "click" "{}"
```

Comment:
I've separated this `msgFor` on a module called `Events` because `Query` was already a little big and it didn't seem like this was a thing that `Query` should do